Common/include/CConfig.hpp

```diff
@@ -4225,6 +4228,24 @@ class CConfig {
    */
   su2double GetRelaxation_Factor_CHT(void) const { return Relaxation_Factor_CHT; }
 
+  /*!
+   * \brief Get the under-relaxation for flow variables density and energy.
+   * \return under-relaxation for flow variables.
+   */
+  su2double GetUnderRelax_Flow(void) const { return Max_Update_Flow; }
+
+  /*!
+   * \brief Get the under-relaxation for SA variable, nu_tilde.
+   * \return under-relaxation for SA variables.
+   */
+  su2double GetUnderRelax_SA(void) const { return Max_Update_SA; }
+
+  /*!
+   * \brief Get the under-relaxation for SST turbulence variables k and omega.
+   * \return under-relaxation for SST variables.
+   */
+  su2double GetUnderRelax_SST(void) const { return Max_Update_SST; }
```

Comment:
Address the names of these functions please, as I mentioned these are not relaxation factors.
MaxUpdateFraction is more consistent.
Please remove underscores from the names of these variables and functions. We've put underscores in arbitrary places in the past and it just makes the code ugly :sweat_smile: